StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/RoundedIconView.swift

```diff
@@ -46,10 +49,10 @@ final class RoundedIconView: UIView {
         switch image {
         case .image(let image):
             iconImageView.image = image.makeImage(template: true)
-        case .imageUrl(let imageUrl):
+        case .imageUrl(let imageUrl, let fallbackImage):
```

Comment:
another thing to note (excluding naming), focusing on functionality (mostly), placeholder is an image that appears BEFORE the real image appears

so "fallbackImage" might be misleading (it makes it seem like its a fallback to errors [which is the original problem why this PR was created])

if, for example, the request takes a long time to load, the "placeholder" will keep showing (which may or may not be expected); in this case maybe it does not matter, but if there's a big difference between "placeholder" and "real image from backend" that might be jarring

https://github.com/stripe/stripe-ios/blob/aa04abcd/StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/UIImageView+Extensions.swift#L20

Google Gemini definition of placeholder: "A placeholder image is a temporary image that's used to indicate where an actual image will be placed in a design."

that does fit what the "image loading helper/library" does

Reply:
It's a good point - I renamed the field to `placeholder` to make this more clear. In terms of the implementation, the reason this works is because the imageview's image is never updated from the placeholder if there is an error loading the remote image. In terms of the possible jarring switch to the "real" image - I think this is standard behavior for a remote image view, like you said!